templates/vue/.vscode/settings.json

```diff
@@ -11,5 +11,8 @@
   ],
   "eslint.autoFixOnSave": true,
   "editor.formatOnSave": false,
-  "vetur.validation.template": false
+  "vetur.validation.template": false,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
```

Comment:
This was an automated change due to the old setting (`eslint.autoFixOnSave`) being deprecated.